admin/app/utils.py

```diff
@@ -64,13 +64,10 @@ def authenticate():
         {'WWW-Authenticate': 'Basic realm="Login Required"'})
 
 
-def requires_auth(f):
-    @wraps(f)
-    def decorated(*args, **kwargs):
-        if not check_auth(current_app.config, request.authorization):
+def requires_auth():
+    if not check_auth(current_app.config, request.authorization):
+        if request.endpoint != 'main.static_logo':
```

Comment:
could we exit early here?

```
def requires_auth():
    if request.endpoint == 'main.static_logo':
        return True

    if not check_auth(current_app.config, request.authorization):
        return authenticate()
```

it's a small thing but it shows the intent that we always want to display static logo regardless of the auth situation

Reply:
Sure